com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/broadcom/lsp/cdi/module/service/ServiceModule.java

```diff
@@ -66,6 +66,7 @@ private void bindCompletions() {
     completionBinding.addBinding().to(ParagraphCompletion.class);
     completionBinding.addBinding().to(SnippetCompletion.class);
     completionBinding.addBinding().to(KeywordCompletion.class);
+    completionBinding.addBinding().to(CopybookCompletion.class);
```

Comment:
This should be duplicated in the TestModule

Reply:
Added